### ISSUE TITLE
No error if authentication fails

### ISSUE THREAD

**Issue by robin13** [elastic/elasticsearch-support-diagnostics#59]
If Shield is enabled on the cluster and the user forgets to use the parameters -c and -p (username/password) the support-diagnostics completes without errors, but every file contains only:

```
{
  "error" : "AuthenticationException[missing authentication token for REST request [/?pretty]]",
  "status" : 401
}
```

Need to add testing to ensure that authentication is used.

Cheers,
-Robin-

**Comment by inqueue** [comment]
If credentials are not provided and authentication is needed, support-diagnostics should catch the 401 and exit prior to creating the output directory and files within it.

**Comment by jjfalling** [comment]
Should be fixed in the 2.x release. If not please reopen.
